### DISCUSSION TITLE
Can Repo "Last Update" sort have a filter option so that it doesn't show updates when made by dependabot?

### DISCUSSION THREAD

**Discussion by Netchicken** [community/community#27246] [Code Security]
Or more precisely, My repositories are out of order of when I have been working on them because Dependabot changes mark the repository as updated.  As a result old repos end up at the top of my list, because they are updated more,  making it hard for me and others to find what I am recently working on. 

I still want Dependabot to update the repositories, just not having those updates showing when repos are sorted by Last updated.

Thanks

**Comment by caraar12345** [comment]
I am so glad someone else has said exactly what I was thinking 😅

**Comment by jfullstackdev** [comment]
you can create a List for that, so it will be always there

https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars

I don't know what specific situation you are in, for you to require too much projects that should be seen in a single time, for example, I was assigned in a certain project almost 4 months ago and it is still the same project, the only repo I'm concerned with, despite the company's other projects, so I just bookmarked it and I don't need to see other projects

it was almost same question about GitHub Desktop, when a developer needs to switch too many projects in a single time, nope, no one can actually do that, that's why GitHub Desktop just save the last repo and when the user opened it, it's the same last repo

**Reply by Netchicken** [reply]
Sorry for the time delay, I had given up on getting any replies to this issue :-) I make lots of small projects because I am a software teacher, and each one is either for students or assessments. Because I have about 70 repos trying to find one is a pain when they are all in a random order because of dependabot.
